ansible/ansible-ai-connect-service
# PR to update happy-dom and related package in Ansible AI connect chatbot

Jira Issue: NA


## Description

PR to update happy-dom and related package in Ansible AI connect chatbot

## Testing

### Steps to test
1. Pull down the PR
2. Run npm audit
3. Critical vulnerability should be 0

### Scenarios tested

NA

## Production deployment
